playground/src/components/view_mode/evaluation_mode/configure_modules.tsx

```diff
@@ -0,0 +1,469 @@
+  const slide = (direction: "prev" | "next") => {
+    if (!scrollRef.current) return;
+    const moveAmount = scrollRef.current.clientWidth * 0.4;
+    const slider = scrollRef.current;
+
+    if (
+      direction === "next" &&
+      slider.scrollLeft < slider.scrollWidth - slider.clientWidth
+    ) {
+      slider.scrollBy({ left: moveAmount, behavior: "smooth" });
+    } else if (direction === "prev" && slider.scrollLeft > 0) {
+      slider.scrollBy({ left: -moveAmount, behavior: "smooth" });
+    }
+  };
+
+  useEffect(() => {
+    checkScroll();
+  }, [moduleConfigurationsState]);
+
+  useEffect(() => {
+    if (!scrollRef.current) return;
+    const scroll = scrollRef.current;
+    scroll.addEventListener("scroll", checkScroll);
+    return () => scroll.removeEventListener("scroll", checkScroll);
+  }, [scrollRef]);
+
+  const configurationsStatus = moduleConfigurationsState.map(
+    (moduleConfiguration) => {
+      const isModuleEmpty = moduleConfiguration.moduleAndConfig === undefined;
+      const isDuplicateName =
+        moduleConfigurationsState.filter(
+          (currentModuleConfiguration) =>
+            currentModuleConfiguration.name === moduleConfiguration.name
+        ).length > 1;
+      const isNameEmpty = moduleConfiguration.name === "";
+      return {
+        id: moduleConfiguration.id,
+        isValid: !isModuleEmpty && !isDuplicateName && !isNameEmpty,
+        isModuleEmpty,
+        isDuplicateName,
+        isNameEmpty,
+      };
+    }
+  );
+
+  const handleExport = () => {
+    moduleConfigurationsState.forEach((config, index) => {
+      setTimeout(() => {
+        const blob = new Blob([JSON.stringify(config, null, 2)], {
+          type: "text/json",
+        });
+        const url = window.URL.createObjectURL(blob);
+        const link = document.createElement("a");
+        link.href = url;
+        let name = config.name.toLowerCase().replace(/\s+/g, "_");
+        name = name.replace(/[\\/:"*?<>|]+/g, "").replace(/_+/g, "_");
+        link.download = `${experiment.exerciseType}_module_config_${name}.json`;
+        link.click();
+        window.URL.revokeObjectURL(url);
+      }, index * 1000);
+    });
+  };
+
+  return (
+    <div className="bg-white rounded-md p-4 mb-8 space-y-2">
+      <div className="flex flex-row justify-between items-center">
+        <div className="flex flex-row items-center gap-2">
+          <h3 className="text-2xl font-bold">Configure Modules</h3>
+          {configurationsStatus.some((status) => !status.isValid) && (
+            <span className="rounded-full bg-red-500 text-white px-2 py-0.5 text-xs">
+              {configurationsStatus.filter((status) => !status.isValid).length}{" "}
+              Invalid
+            </span>
+          )}
+        </div>
+        <div className="flex flex-row gap-2">
+          <button
+            disabled={
+              configurationsStatus.some((status) => !status.isValid) ||
+              configurationsStatus.length == 0
```

Comment:
```suggestion
              configurationsStatus.length === 0
```
Should not matter here, but I think always using `===` is better